website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -28,8 +28,8 @@ func test_integers{syscall_ptr : felt*, range_check_ptr}(a : felt, b : felt) {
 ```
 
 :::note
-By default, Protostar applies the [`strategy.felts()`](#strategyfelts) strategy to all felt
-parameters.
+It is required to either provide a strategy using [`given`](../02-cheatcodes/given.md)
+or specific values for parameters using [`example`](../02-cheatcodes/example.md).
```

Comment:
This is not a properly delivered information. If `given` is necessary then we shouldn't put it in a note but into main section. Please remove this note.

Reply:
sure

Reply:
[docs](https://github.com/software-mansion/protostar/pull/995/commits/ece63bb97c024245dd553f1c0720d0305c96b92c)